cmake/FindPython2.cmake

```diff
@@ -0,0 +1,171 @@
+# Distributed under the OSI-approved BSD 3-Clause License.  See accompanying
```

Comment:
py2 requirement has been removed from GUI, is there a need to reintroduce it for cmake? Cant it used py3?

Reply:
Removed, and disabled Qt4 and py2 from cmake building - I guess it will be superceded by py3 and Qt5 soon anyway.